app/src/main/res/layout/activity_metadata.xml

```diff
@@ -158,13 +158,49 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/metadata_species_title" />
 
+            <TextView
+                android:id="@+id/metadata_comments_title"
+                style="@style/TextView.SectionTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="24dp"
+                android:layout_marginEnd="16dp"
+                android:text="@string/metadata_comments_title"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.0"
+                app:layout_constraintTop_toBottomOf="@+id/metadata_species_spinner"
+                app:layout_constraintVertical_bias="0.0" />
+
+            <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/metadata_comments_layout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="16dp"
+                android:textColorHint="@color/colorPlainText"
+                app:helperText="@string/metadata_comments_layout_hint"
+                app:helperTextEnabled="true"
+                app:helperTextTextColor="@color/colorPlainText"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/metadata_comments_title">
+
+                <EditText
+                    android:id="@+id/metadata_comments_input"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:ems="10"
+                    android:inputType="text"
+                    android:hint="@string/metadata_comments_input_hint" />
+            </com.google.android.material.textfield.TextInputLayout>
```

Comment:
Instead of using a `TextView` for the section title, and then a hint as `Input text` for the `TextInputLayout` (which personally, I think that it brings no value, as it doesn't tell you what the field is about), why not using the section title as the hint directly, and drop the `TextView`? This is just my opinion, but I would like to hear your thoughts. Maybe also run it with the designer too or someone else :D

Reply:
The reason I wrapped the `EditText`is because I needed to add the helper text `Please, don't include any sensitive information.`.

I'm not sure if I understand your proposal. Do you mean moving the hint `Input text` and replacing it with the helper text `Please, don't include [...]`? Doing it like this we could remove the `TextInputLayout`. I agree that the `Input text` field does not bring any value.

Reply:
I mean the other way around, using the section title `@string/metadata_comments_title` as the hint. Something like this:
```suggestion
            <com.google.android.material.textfield.TextInputLayout
                android:id="@+id/metadata_comments_layout"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_marginStart="16dp"
                android:layout_marginEnd="16dp"
                android:textColorHint="@color/colorPlainText"
                app:helperText="@string/metadata_comments_title"
                app:helperTextEnabled="true"
                app:helperTextTextColor="@color/colorPlainText"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintHorizontal_bias="0.0"
                app:layout_constraintTop_toBottomOf="@+id/metadata_species_spinner"
                app:layout_constraintVertical_bias="0.0">

                <EditText
                    android:id="@+id/metadata_comments_input"
                    android:layout_width="match_parent"
                    android:layout_height="match_parent"
                    android:ems="10"
                    android:inputType="text"
                    android:hint="@string/metadata_comments_input_hint" />
            </com.google.android.material.textfield.TextInputLayout>
```

Reply:
Oh, I see, thanks for the clarification.

I applied it to see how it looks. I swapped the text, I think the helper looks good at the bottom


<img src="https://user-images.githubusercontent.com/44713767/89870557-5122fe00-dbb6-11ea-804a-32de69309b73.png" width="300">